Makefile

```diff
@@ -0,0 +1,22 @@
+DOCKER_REPO=quay.io/kinvolk
+
+IMAGES=$(shell (ls -d */ | cut -d/ -f1))
```

Comment:
This is a bit ugly but I don't know of a better solution

Reply:
Me neither :smile: `make` wizards, help welcome!